vital/bindings/python/vital/modules/module_loader.py

```diff
@@ -49,7 +49,7 @@ def _load_python_module(mod):
                 logger.warn(('Python module "{}" defined {} but '
                              'it is not callable').format(mod, registrar))
 
-    logger.warn(('Python module "{}" does not have registrar method').format(mod))
+    logger.debug(('Python module "{}" does not have registrar method').format(mod))
```

Comment:
I could see that this warning is useful in determining why your module will not load. If there are too many of these messages, is there another way to limit them such as moving the non-loadable python modules to a different directory?

Reply:
Agreed that this warning is useful. One of the changes that was made in the wheel was to start specifying python modules explicitly rather than searching for modules in a package. This is mainly being used for python plugin registration that is being done using entry-point. Maybe we can impose the same requirement for SPROKIT_PYTHON_MODULES based registration.